web-platform/functions/src/index.ts

```diff
@@ -13,6 +13,8 @@ admin.initializeApp({
   credential: admin.credential.applicationDefault(),
 });
 
+const OFFBOARD_URL = "https://rally.mozilla.org/leaving-rally/index.html";
```

Comment:
This will need to change obviously but it's in our infra not the extension, so I think it's probably OK as a placeholder. We should aim to have this replaced before the extension goes live.